pom.xml

```diff
@@ -14,11 +14,11 @@
     <packaging>atlassian-plugin</packaging>
 
     <properties>
-        <bitbucket.version>5.5.0</bitbucket.version>
+        <bitbucket.version>6.0.1</bitbucket.version>
```

Comment:
Does not this require increase of plugin major version in pom.xml and documenting it in README.adoc?

Reply:
The project owner should do that but he is not responding.